wp-content/themes/core/components/header/subheader/Controller.php

```diff
@@ -4,12 +4,20 @@
 namespace Tribe\Project\Templates\Components\header\subheader;
 
 use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Traits\Page_Title;
 
 class Controller extends Abstract_Controller {
+	use Page_Title;
 
-	public function title(): string {
-		// TODO: implement a wrapper/container component
-		// TODO: use the method in head/controller.php?
-		return get_the_title();
+	public function render_title(): string {
```

Comment:
I'm not sure I like `render_title()` as a name. I much prefer the previous `title()` or `get_title()` as it doesn't render anything unless it's echoed.

Reply:
Fair enough, noting that this is something we should do a consistency pass on once we are wrapped with the getting everything working pivot work as I think this is what we or I have done thus far : )